integrations/webchat/agent-assist/agent-verify/README.md

```diff
@@ -0,0 +1,188 @@
+
+After you provide a name for your widget on the Details tab, you will need to navigate to the Configuration tab. From here you will need to configure both the `Application URL`, the `Iframe Sandbox Options` and `Group Filtering`.
+
+#### Application URL
+The `Application URL` should take the following form:
+
+```
+http://<webserver>/genesys-agent-assist-verify.html?clientID=<genesys client id>&environment=mypurecloud.com&conversationID={{pcConversationId}}
+```
+`webserver`: host address and port for the webserver that is serving up the client application files included in this sample application.
+
+`clientID`: generated when you create the OAuth credentials to be used for the implicit token login (see below).
+
+`environment`: based on the location of your Genesys account. The environment shown works for North American accounts.
+
+`conversationID`: generated by Genesys at runtime and the field will be populated by the agent application.
+
+#### Iframe Sandbox Options
+The `Iframe Sandbox Options` should be filled in with these options:
+
+```
+allow-scripts,allow-same-origin,allow-forms,allow-modals,allow-popups,allow-presentation,allow-downloads
+```
+#### Group Filtering
+The `Group Filtering` section is selected from a pre-configured group that can be created from the Admin page under Directory->Groups. You need to make sure that all agents that require the new agent-verify widget are listed within the chosen group.
+
+### Step 8: Configure OAuth for the Interaction Widget
+
+![Genesys Interaction Widget Config Page](images/oauth-client-page.png)
+Start by going to `OAuth` under Integrations and it `Add Client` to create a new OAuth client.
+
+![Genesys Interaction Widget Config Page](images/oauth-client-details-top-page.png)
+Here you'll need to configure the App Name associated with the credentials, the Grant Type which must be `Token Implicit Grant (Browser)`, the redirect URL which is shown below:
+```
+http://localhost:8080/genesys-agent-assist-verify.html?clientID=xxxxx&environment=mypurecloud.com
+```
+and the Scope which is set to `notifications`. Note that the redirect URL does not need to include the conversationID.
+
+![Genesys Interaction Widget Config Page](images/oauth-client-details-bottom-page.png)
+Once the previous page is saved, you will be presented with more details about your OAuth configuration including the Client ID which is needed when configuring the Interaction Widget.
+
+### Step 9: Setup Speech and Text Analytics
+
+![Genesys Speech Analytics Page](images/speech-analytics-page.png)
+
+Navigate to the `Speech and Text Analytics` page from the Admin page.
+
+![Genesys Speech Analytics Config Page](images/speech-analytics-config-page.png)
+
+On this page you will need to enable voice transcriptions and `Low Latency Transcription`. You can read more about the settings on this page [here](https://help.mypurecloud.com/articles/speech-and-text-analytics).
```

Comment:
FYI, it is considered bad practice for accessibility to use "here" links. Basically the idea is that if the user selects the link with their screenreader, they don't really know what it does because the text just says "here". The link should really be the text of where it is going to.
```suggestion
On this page you will need to enable voice transcriptions and `Low Latency Transcription`. You can read the [speech setting documentation](https://help.mypurecloud.com/articles/speech-and-text-analytics) for more information.
```